pkg/components/imageregistry/component.go

```diff
@@ -33,9 +33,15 @@ var ImageRegistryComponent = Component{
 			{
 				SIG: "sig-imageregistry",
 			},
+			{
+				IncludeAny: []string{
+					":ImageRegistry ",
+				},
+				Priority: 1,
+			},
 			{Suite: "ImageStream Manifest"},
 			{Suite: "Image_Registry"},
-			{Suite: "remote registry related scenarios"},
+			// {Suite: "remote registry related scenarios"},
```

Comment:
Why is this commented out? If it's unneeded it should just get removed

Reply:
Removed this line.